.github/workflows/python_bindings.yml

```diff
@@ -206,9 +212,21 @@ jobs:
         python setup.py bdist_wheel
         python -m twine upload --skip-existing --repository testpypi dist/*
 
+    - name: Zip the wheel to maintain case sensitivy and file permissions
+      working-directory: ./build/Products/python_package/
+      shell: bash
+      run: |
+        tar -cvzf openstudio-${{ matrix.name }}.tar.gz dist/
+
+    - name: Upload .whl to artifact
+      uses: actions/upload-artifact@v2
+      with:
+        name: openstudio-${{ matrix.name }}  # something like openstudio-Windows_py37
+        path: ./build/Products/python_package/openstudio-${{ matrix.name }}.tar.gz
```

Comment:
Use artifact upload to pass information between jobs. The build uploads, the test job downloads

Reply:
Pretty clever use of artifacts here. 👍